UInt32MultiArray img_rgb
---
UInt32MultiArray img_gray
